app/assets/stylesheets/ama_layout/layout_components/sidebar.scss

```diff
@@ -117,7 +111,7 @@ span.nav-icon{
   margin: $base-margin*2;
   margin-top: $base-margin*4;
   font-size: $base-font-size*1.5;
-  color: $smoke;
+  color: rgb(179, 179, 179);
```

Comment:
Is $smoke too light for this? If we want to make a new grey shade we should probably make a new colour variable for it.